assets/js/project.js

```diff
@@ -1,7 +1,18 @@
-jQuery( function( $ ) {
-	// $( 'p' ).css( 'background-color', 'blue' );
+// Initialize Foundation
+jQuery( document ).ready( function( $ ) {
```

Comment:
good use of indentation.  I also like that you wrapped both javascript library inits in jquery ready.